client/components/Home.js

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import { Redirect } from "react-router-dom"
+
+const Home = (props) => {
+    
+    return(
+       <Redirect to="/pets" />
```

Comment:
I would keep this component if it will be used for something in the future, but if it is just redirecting we can handle this in other ways